lighthouse-core/report/html/templates.html

```diff
@@ -366,7 +366,7 @@
     <a href="" class="lh-topbar__url" target="_blank" rel="noopener"></a>
 
     <div class="lh-tools">
-      <button class="report-icon report-icon--share lh-tools__button" title="Export report">
+      <button class="report-icon report-icon--share lh-tools__button" title="Tools menu" aria-label="Toggle report tools menu">
```

Comment:
at what point does i18n creep into here? :)

Reply:
> at what point does i18n creep into here? :)

😬 

(probably should have a while ago)

Reply:
this be the point: #9166